### DISCUSSION TITLE
3.23.1a PTU.9188344 Überlappende Texte

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#555] [Übersetzungsvorschläge]
Wegen Star Citizen Neuinstallation habe ich mir diese Version 3.23.1a PTU.9188344 heute in das LIVE gezogen und überlappende Texte gefunden. In der Prospector und in der C2 Starlifter überlappen Texte sich mit der Treibstoffanzeige. 2 Schiffe sind auf jeden Fall betroffen.

![ScreenShot-2024-06-03_20-18-50-21D](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/38e6d6e7-98d2-4dfd-8079-b6f026b964b9)

![ScreenShot-2024-06-03_20-02-31-EF0](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/595d8f4a-4a94-477a-9707-0c01d02b5ffe)

In der Konsole der Raffinerie überlappt sich der Schiffsname von der C2 mit der SCU Zahl. Vielleicht würde hier Crusader C2 Starlifter ausreichen. Wenn man diese Texte ändern könnte.

![ScreenShot-2024-06-03_20-12-21-0D0](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/c343d899-e2bb-49c4-892e-2941f8930022)

**Comment by rjcncpt** [comment]
Hi.

Also zu `Benötigter Treibstoff` habe ich keine wirkliche Übersetzung die kürzer ist außer nur "Kraftstoff". (Bei uns ist alles Kraftstoff der Einfachheit halber)

Zu `Ausgerichtet` gibt es auch nichts kürzeres außer "Syched" für Synchronisiert. Das wäre ja aber wieder english. Wohl aber zu verschmerzen.

Zur "Starlifter" kann man zwar den Text ändern, halte ich aber nicht für Sinnvoll da es dann wohl Global geändert ist. Das ist ein CIG Problem.

**Comment by BTSVDanielk** [comment]
Kraftstoff hört sich besser an als Treibstoff. Dann könnte man die überdeckte Zahl wieder sehen.
Zu Ausgerichtet habe ich noch nichts.

Das mit Starlifter habe ich jetzt auch verstanden ;).

**Reply by BTSVDanielk** [reply]
Guten Morgen, ich poste noch etwas in diese Diskussion, ist ziemlich ähnlich vom Thema. Soll das in den Raffinerieaufträgen so sein? Oder hat das C vor dem SCU etwas zu bedeuten, wovon ich nicht weiß?

![ScreenShot-2024-06-05_06-22-42-7B9](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/6c5ed0fb-c60e-4891-8d00-1065e2a123d5)

**Reply by BTSVDanielk** [reply]
Über die verbleibende Zeit, wo der ertrag in SCU steht, ist vor SCU auch ein C.

**Reply by BTSVDanielk** [reply]
soll das ganze für mikro SCU stehen ?

**Reply by rjcncpt** [reply]
> soll das ganze für mikro SCU stehen ?

Hey. Ich habe mal geschaut. Das ist die fragwürdige CIG Entscheidung, alles in CAPS zu schreiben. In der ini Datei steht (cSCU).

:)
